Packs/Gurucul/ReleaseNotes/1_1_0.md

```diff
@@ -1,45 +0,0 @@
-##Gurucul Integration
-In this release Below points covered
->##IncidentFields
->   Below are the incident fields
->
-    1.Gra Case Id
-    2.Gra Entity
-    3.Gra Entity Type
-    4.Gra open Date
-    5.Gra owner
-    6.Gra Owner Name
-    7.Gra Owner Type
-    8.Gra Risk Date
-    
->##Incident Types
->    Below are the 2 incident types
->
-    1.GraCases
-    2.GraHighRiskUser
->##Commands
->
-| **Command Name** | **Description** | **Arguments(comma separated)** |
-| --- | --- | --- | 
-|gra-fetch-users|Retrieves list of all Users (Identities)|page,max|
-|gra-fetch-accounts|Retrieves all Accounts information|page,max|
-|gra-fetch-active-resource-accounts|Retrieve list of all Active Accounts for a given resource.|resource_name,page,max|
-|gra-fetch-user-accounts|Retrieve list of all Active Accounts and details for a given User / Identity|employee_id,page,max|
-|gra-fetch-resource-highrisk-accounts|Retrieve High Risk Accounts for a given Resource|resource_name,page,max|
-|gra-fetch-hpa|Retrieve list of all High Risk Privileged Accounts|page,max|
-|gra-fetch-resource-hpa|Retrieve all High Privileged Accounts for a given Resource|resource_name,page,max|
-|gra-fetch-orphan-accounts|Retrieve list of all Orphan / Rogue Accounts|page,max|
-| gra-fetch-resource-orphan-accounts|Retrieve all Orphan / Rogue Accounts for a given Resource|resource_name,page,max|
-|gra-user-activities|Retrieve User activity for a given User|employee_id,page,max|
-|fetch-gra-users-details|Retrieve User details for a given user|employee_id,page,max|
-|gra-highrisk-users|Retrieve all High Risk User information|page,max|
-|gra-cases|Retrieve list of all Cases for a given Status|status|
-|gra-user-anomalies|Retrieve Behavioral Anomalies triggered for a given User|employee_id|
-
->##Playbooks
-> Below are then 2 playbooks
->
-    1.GraHighRiskUsers
-    2.GraUsersCases
-    
-    
```

Comment:
Was this removed by accident?

Reply:
suggestion as per circleCI app , removed
but as per your suggestion again I will add to repository